src/app.js

```diff
@@ -243,20 +243,7 @@ class SAFEApp extends EventEmitter {
   * @param {String} state
   */
   set networkState(state) {
-    switch (state) {
```

Comment:
We need to have this conversion from a `number` to the textual representation somewhere, if it's not done from this function, then from the `get networkState()`: https://github.com/maidsafe/safe_app_nodejs/blob/master/src/app.js#L266

Reply:
This is also affecting/changing current behaviour of the notifications, and that would break the current applications acting on the string returned in the notifications.